doc/go-target.md

```diff
@@ -0,0 +1,14 @@
+# ANTLR4 Language Target, Runtime for Go
+
+### Getting started
+
+1. Get the runtime and install it on your GOPATH: `go get github.com/antlr/antlr4`
+2. Generate the parser/lexer code: `antlr MyGrammar.g4 -Dlanguage=Go`
+
+### Referencing in your code
+
+Reference the go package like this:
```

Comment:
Would it be confusing whether "go package" refers to the runtime package or the generated code here?

Do we need to call this out explicitly when the generated code refers to it?